app/eventyay/base/payment.py

```diff
@@ -1375,6 +1389,10 @@ def payment_prepare(self, request: HttpRequest, payment: OrderPayment) -> Union[
                     'Please contact the organizer of this event.'
                 ),
             )
+        except Exception as e:
```

Comment:
Don't use the general `Exception` class. There may be some bugs from developers mistake that needs to be exposed early to let developers fix.